CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 1.80.1
```

Comment:
don't know about the minor/major conventions you use..

Reply:
If minor change you can update middle number otherwise if only bugfix, the last number ;)

Reply:
since a new feature is added with this release, i'd bump the minor version (number in the middle).